cli/service/service.go

```diff
@@ -76,6 +81,103 @@ func List() {
 	fmt.Printf("%s\n", jsonBytes)
 }
 
+func Log(ServiceName string, ServiceNameTail string) {
+	msgPrinter := i18n.GetMessagePrinter()
+	RefURL := ServiceName
+	if ServiceNameTail != "" {
+		RefURL = ServiceNameTail
+	}
+	if ServiceNameTail == "" && ServiceName == "" {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, i18n.GetMessagePrinter().Sprintf("No service name inserted."))
+	}
+	var inputs api.AllServices
+	cliutils.HorizonGet("service", []int{200}, &inputs, false)
+	id_found := 0
+	var instance_id string
+	for i := 0; i < len(inputs.Instances["active"]); i++ {
+		a, err := json.Marshal(inputs.Instances["active"][i])
+		if err != nil {
+			cliutils.Fatal(cliutils.JSON_PARSING_ERROR, i18n.GetMessagePrinter().Sprintf("Unable to read Service %v.", RefURL))
+		}
+		var id api.MicroserviceInstanceOutput
+		json.Unmarshal((a), &id)
+		if strings.Contains(id.SpecRef, RefURL) {
+			instance_id = id.InstanceId
+			id_found = 1
+			break
+		}
+	}
+	if id_found == 0 {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, i18n.GetMessagePrinter().Sprintf("Service %v is not running on the node.", RefURL))
+	}
+	fmt.Printf("Displaying syslog messages for service %v with service id %v. Use ctrl-C to terminate this command.\n", RefURL, instance_id)
```

Comment:
RefURL is not the right variable to use here. I think it should be the id.SpecRef value.